Sources/TuistGenerator/Models/Scheme.swift

```diff
@@ -0,0 +1,132 @@
+import Basic
+import Foundation
+
+public class Scheme: Equatable {
```

Comment:
It's a shame - git didn't recognize this as a moved file

Reply:
If the code inside has changed git will struggle to keep the reference, you could give it a helping hand and use `git mv` [ref](https://git-scm.com/docs/git-mv)